sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleDataView.java

```diff
@@ -156,7 +156,8 @@ protected void initView(String params) {
 		final boolean deleted = FacadeProvider.getSampleFacade().isDeleted(uuid);
 
 		if (deleted) {
-			editComponent.setEditable(false, CommitDiscardWrapperComponent.DELETE_UNDELETE);
+			editComponent.addButtonToExcludedList(CommitDiscardWrapperComponent.DELETE_UNDELETE);
+			editComponent.setEditable(false, editComponent.getExcludedButtons().stream().toArray(String[]::new));
```

Comment:
would be better to have a method in CommitDiscard component tthat disablesIt with the excluded buttons so there would be no need to expose `getExcludedButtons` and here you just would call something like `editComponent.setNonEditable()`